lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -401,7 +403,7 @@ def parse(body)
 
       def commit(action, parameters, options)
         begin
-          raw_response = ssl_post("#{url}/#{action}", post_data(action, parameters), request_headers(options))
+          raw_response = ssl_post("#{base_url(action)}/#{action}", post_data(action, parameters), request_headers(options))
```

Comment:
Would it improve readability to have `base_url` return the complete URL including the action endpoint?

```ruby
raw_response = ssl_post(base_url(action), post_data(action, parameters), request_headers(options))
```